server/data/orchestrationApiClient.ts

```diff
@@ -168,6 +168,9 @@ export default class OrchestrationApiClient {
     return this.restClient.get({ path: `/visits/notification/visit/${reference}/types` })
   }
 
+  // orchestration-prisons-exclude-date-controller
+  // TODO put exclude-date calls here
```

Comment:
Looks like add/remove and get past/future are all in the orchestration service now so we can put them all here 👍